### PR TITLE
CLN: isort imports- util dir

### PR DESCRIPTION
**Pull request by alimcmaster1**
Signed-off-by: alimcmaster1 <alimcmaster1@gmail.com>

util directory/.py files in pandas directory.

- [x] xref #23334 
- [ ] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Review by WillAyd — APPROVED** [review]
lgtm if green

**Comment by alimcmaster1** [comment]
Any chance a core dev - could restart the 3.6 build on Travis. ( Or grant me the relevant permission so I can do this myself going forward - happy to discuss this offline if appropriate )

I seem to be getting this:
`The job exceeded the maximum time limit for jobs, and has been terminated`

**Comment by TomAugspurger** [comment]
Restarted. LMK if it times out again and I'll take a closer look.

**Comment by jreback** [comment]
pls merge master. and ping on green.

**Comment by jreback** [comment]
needs another merge of master

File: pandas/util/testing.py
Comment (jreback) [inline pandas/util/testing.py]: why is this here?

File: pandas/lib.py
Comment (jreback) [inline pandas/lib.py]: can you remove the deprecated ones they are being removed elsewhere

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jreback** [comment]
can you merge master, we just updated the isort cfg

**Comment by alimcmaster1** [comment]
ping - @jreback merged master and now green

**Review by jreback — APPROVED** [review]

**Pull request merged**
